intelmq/lib/bot.py

```diff
@@ -241,6 +243,9 @@ def catch_shutdown():
         if start:
             self.start()
 
+    def harmonization(self):
+        return self.__harmonization
+
```

Comment:
Just an idea: Why not use [`@property`](https://docs.python.org/3/library/functions.html#property)